config/samples/featuregates.yaml

```diff
@@ -7,4 +7,8 @@ data:
   # timeTravel allows navigating through cluster states across time.
   # It is useful for scenarios where you want to view historical data or
   # predict future states based on current trends. Default is "false".
-  # timeTravel: "false"
+  timeTravel: "false"
```

Comment:
Do you prefer to keep all the featuregates commented out in the sample file ?
The reason I uncommented it were to prevent formatting errors in the IDE
```
Incorrect type. Expected "object".yaml-schema: kubernetes://schema/v1%40configmap
```

Reply:
In that case, since we're adding actual gated features already, maybe it would be best to get rid of the dummy feature as the first commit of this series?

As for the contents of the sample file, admittedly I'm not sure how it's used.  If it's an actual sample for users to use then I assume its contents should ideally match the defaults the controller uses.

Reply:
> In that case, since we're adding actual gated features already, maybe it would be best to get rid of the dummy feature as the first commit of this series?

I agree..